client/validators.py

```diff
@@ -0,0 +1,32 @@
+import re
+from django.core.exceptions import ValidationError
+
+ID_NUMBER_REGEX = r'^\d{8,9}$'
+
+
+def id_number_validator(value):
```

Comment:
This function was copied from client/forms.py with very minor changes (to docstring and comment)